plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/PinotPageSourceProvider.java

```diff
@@ -104,4 +115,21 @@ public ConnectorPageSource createPageSource(
         }
         throw new UnsupportedOperationException("Unknown Pinot split type: " + pinotSplit.getSplitType());
     }
+
+    static Optional<String> getQueryOptionsString(String options)
+    {
+        if (isNullOrEmpty(options)) {
+            return Optional.empty();
+        }
+
+        Map<String, String> queryOptionsMap = ImmutableMap.copyOf(MAP_SPLITTER.split(options));
+        if (queryOptionsMap.isEmpty()) {
+            return Optional.empty();
+        }
+        String result = queryOptionsMap.entrySet().stream()
+                .filter(kv -> !Strings.isNullOrEmpty(kv.getKey()) && !Strings.isNullOrEmpty(kv.getValue()))
+                .map(kv -> CharMatcher.anyOf("\"`'").trimFrom(kv.getKey()) + "=" + kv.getValue())
+                .collect(Collectors.joining(",", " option(", ") "));
```

Comment:
OPTION keyword is deprecated. I disagree with supporting it.
https://docs.pinot.apache.org/users/user-guide-query/query-options#option-keyword-deprecated

Reply:
@ebyhr Made the changes to use SET key='value'; format.